src/context/index.tsx

```diff
@@ -14,7 +14,10 @@ interface Props {
 export const AppContextProvider = ({ children }: Props) => {
   const [darkmode, setDarkmode] = useState<boolean>(false);
 
-  const contextValues = { darkmode, setDarkmode };
+  const storedDarkmode = localStorage.getItem('darkmode');
+  const initialDarkmode = storedDarkmode ? JSON.parse(storedDarkmode) : false;
```

Comment:
You aren't storying this as the "initial" state, as it is intended to be used.
The 'darkmode' value is not not used, and the 'setDarkmode' value would never have an effect on the "contextValues's darkmode, as it is set to "initialDarkmode'